bodhi/util.py

```diff
@@ -435,12 +435,10 @@ def request2html(context, request):
 def update2html(context, update):
     request = context.get('request')
 
-    if hasattr(update, 'title'):
-        title = update.title
-    else:
-        title = update['title']
```

Comment:
I think this was supposed to handle the case where ``update`` could be either a sqlalchemy ``Update`` object, or a ``dict``.